src/main/java/io/debezium/connector/jdbc/JdbcSinkConnector.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.jdbc;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.connect.connector.Task;
+import org.apache.kafka.connect.sink.SinkConnector;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.annotation.Immutable;
+
+public class JdbcSinkConnector extends SinkConnector {
+    private static final Logger LOGGER = LoggerFactory.getLogger(SinkConnector.class);
+
+    public JdbcSinkConnector() {
+    }
+
+    @Immutable
+    private Map<String, String> properties;
+
+    @Override
+    public String version() {
+        return Module.version();
+    }
+
+    @Override
+    public void start(Map<String, String> props) {
+        this.properties = Collections.unmodifiableMap(new HashMap<>(props));
+    }
+
+    @Override
+    public Class<? extends Task> taskClass() {
+        return JdbcSinkConnectorTask.class;
+    }
+
+    @Override
+    public List<Map<String, String>> taskConfigs(int maxTasks) {
+        LOGGER.info("Setting task configurations for {} workers.", maxTasks);
+        final List<Map<String, String>> configs = new ArrayList<>(maxTasks);
+        for (int i = 0; i < maxTasks; ++i) {
+            configs.add(properties);
+        }
+        return configs;
```

Comment:
So with this implementation, you'd create the same configuration for _all_ tasks, whereas you'll need to split up the overall workload between tasks. Specifically, each task should work on a subset of the input topic(s), as equally distributed as possible. So say there's ten input topics overall, and maxTasks = 3, then two tasks would take three topics, and one task would take four topics.

Reply:
Totally agree about that but I think for this its enough right now